### DISCUSSION TITLE
What method do you use to count memory consumption?

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#799] [Q&A] [answered]
I tried top and htop command, they can view the memory consumption of the entire computer, but they don't seem to support the statistics of memory by command, when using many mpi processes to run the example, how do you usually calculate the memory consumption?

**Comment by Thomas-Ulrich** [comment, marked as answer]
If you are using slurm, you could do:
```watch -n 1 sstat -j <job_id> --format="JobId,MaxRSS" -P```
